sros2/test/policies/common/node/logging.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<profile>
+  <topics publish="ALLOW" >
+    <topic>/rosout</topic>
```

Comment:
Should this be `<topic>/rosout</topic>`, `<topic>~rosout</topic>` or just `<topic>rosout</topic>`?
I.e. should the topic be left as global, private or relative?

I haven't tested yet, but it looked like from the PR you link that `rosout` might only be needed:

https://github.com/ros2/rcl/pull/350/files#diff-bf8aead5abb376e0ca7e38051163fb9dR34

Reply:
> I haven't tested yet, but it looked like from the PR you link that `rosout` might only be needed:

I actually didnt look in detail at the PR, this change made the access control tutorial work for me while I couldn't get the one on master to work.

> Should this be `<topic>/rosout</topic>`, `<topic>~rosout</topic>` or just `<topic>rosout</topic>`?

Good question, maybe reviewers of  [ros2/rcl#350](https://github.com/ros2/rcl/pull/350) can provide guidance here (@jacobperron @nuclearsandwich)

Reply:
If we namespace a node, e.g.

        ros2 run examples_rclcpp_minimal_publisher publisher_member_function __ns:=/foo

The `rosout` topic does not get pushed into the namespace. So I think the global namespace for `/rosout` makes sense in this context.
Although, I'm not sure if this is intended behavior. I couldn't find any explicit discussion, for or against, keeping a global `/rosout` topic for all nodes (https://discourse.ros.org/t/ros2-logging/6469).

Reply:
Ok, if it doesn't remap, then leaving it global is applicable.

BTW, @jacobperron , if `ROSOUT_TOPIC_NAME` is left unqualified, what keeps that topic from being remapped? Shouldn't `ROSOUT_TOPIC_NAME` be `/rosout`?

https://github.com/ros2/rcl/pull/350/files#diff-bf8aead5abb376e0ca7e38051163fb9dR166

Reply:
That's a good question. Looking at the rosout implementation, it isn't immediately obvious to me why `rosout` is not being remapped...

Reply:
Nevermind. I mistakenly read the output of `ros2 topic list`. `rosout` **does** get remapped in my example to `/foo/rosout`.

Reply:
> Nevermind. I mistakenly read the output of `ros2 topic list`. `rosout` **does** get remapped in my example to `/foo/rosout`.

Good to know, updated accordingly in https://github.com/ros2/sros2/pull/80/commits/f2061f118e018d25fc5dc461445b60b05be826bc